features/google/src/model/importers/utils/lang-constants.js

```diff
@@ -0,0 +1,31 @@
+export const langConstants = {
+    AccessLogImporter: {
+        english: { regex: /\/Access Log Activity\/.*\.csv$/ },
+        german: { regex: /\/Zugriffsprotokollaktivitäten\/.*\.csv$/ },
+        danish: { regex: /\/Aktiviteter i adgangsloggen\/.*\.csv$/ },
+    },
+    ActivitiesImporter: {
+        english: { regex: /\/My Activity\/.*\.html$/ },
+        german: { regex: /\/Meine Aktivitäten\/.*\.html$/ },
+        danish: { regex: /\/Mine aktiviteter\/.*\.html$/ },
+    },
+    SemanticLocationsImporter: {
+        english: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+        german: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+        danish: {
+            regex: /\/[^/]+\/Semantic Location History\/\d+\/[^.]+\.json$/,
+        },
+    },
+};
+export function matchRegex(path, importer) {
+    const importerRegex = langConstants[importer];
```

Comment:
Wouldn't this be `importer.name`? Possibly, by default, constructors are stringified to their name, but `importer` is still an object. Also, if you call this `importer` you can probably just use `this` to invoke it, instead of `this.constructor` (and then `importer.constructor.name` to get the name)